sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/param/ConfigurableRequestItemParser.java

```diff
@@ -0,0 +1,121 @@
+package com.alibaba.csp.sentinel.adapter.gateway.common.param;
+
+import com.alibaba.csp.sentinel.util.StringUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+
+/**
```

Comment:
could you add more description about this class, so that the users can know the purpose and how to use.